src/components/lib/FlowGrid/background-container.js

```diff
@@ -38,13 +48,18 @@ export default class EdgeContainer extends Component {
 
     const rowHeights = this.props.rowHeights
     const containerHeight = _.get(rowHeights, 'length') && rowHeights.reduce((a,b) => a + b)
+
+    if (!columnWidth || !rowHeights.length){ return false }
     return (
-      <Edges
-          columnWidth={columnWidth}
-          containerHeight={containerHeight}
-          edges={edges}
-          rowHeights={rowHeights}
-      />
+      <div>
+        <Edges
```

Comment:
We should note here that we will rename 'Edges' -> 'Arrows' with a comment, so I can find it easily later in a refactor. Shouldn't make that change now, though.